org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/info/InfoWindow.java

```diff
@@ -1066,7 +1067,11 @@ protected String getSQLWhere() {
 		
 		StringBuilder builder = new StringBuilder();
 		MTable table = MTable.get(Env.getCtx(), infoWindow.getAD_Table_ID());
-		if (!hasIsActiveEditor() && table.get_ColumnIndex("IsActive") >=0 ) {
+		MReference ref = m_gridfield != null ? MReference.get(Env.getCtx(), m_gridfield.getAD_Reference_Value_ID()) : null;
```

Comment:
Don't call MReference.get if m_gridfield.AD_Reference_Value_ID is 0. i.e `MReference ref = m_gridfield != null && m_gridfield.getAD_Reference_Value_ID() > 0 ? MReference.get(Env.getCtx(), m_gridfield.getAD_Reference_Value_ID()) : null;
`